### ISSUE TITLE
Port forwarded for ASP.NET service but not shown in Remote Explorer UI

### ISSUE THREAD

**Issue by rgwood** [microsoft/vscode-remote-release#1816]
Issue Type: <b>Bug</b>

Hi, I've noticed something odd about port forwarding in Remote-SSH.

tl;dr: I'm running an ASP.NET Core service that listens on port 5001 in a Linux VM, via VS Code Remote SSH on macOS. I can access the service at localhost:5001 from macOS, which implies that the port is automatically being forwarded. However, the Forwarded Ports pane in the Remote Explorer extension is empty.

## Steps to reproduce:
1. Connect VS Code on Mac to Linux machine
1. In VS Code Linux terminal:
   1. Create ASP.NET service using `dotnet new webapi`
   1. Run service with `dotnet run`
1. In macOS terminal:
   1. Run `curl https://localhost:5001/WeatherForecast --insecure`, observe that the request succeeds even though VS Code is showing nothing in "Remote Explorer" > "Forwarded Ports"

It seems like the Remote-SSH extension is forwarding port 5001 automatically, but not reporting that it is forwarded. Is this a bug or am I misunderstanding something? Thank you!

Extension version: 0.47.2
VS Code version: Code 1.40.0 (86405ea23e3937316009fc27c9361deee66ffbf5, 2019-11-06T17:09:34.601Z)
OS version: Darwin x64 19.0.0
Remote OS version: Linux x64 5.3.0-19-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz (8 x 2700)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 2, 2|
|Memory (System)|16.00GB (0.42GB free)|
|Process Argv||
|Screen Reader|no|
|VM|13%|

|Item|Value|
|---|---|
|Remote|SSH: ubuntu-vm|
|OS|Linux x64 5.3.0-19-generic|
|CPUs|Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz (2 x 2712)|
|Memory (System)|1.93GB (0.63GB free)|
|VM|100%|
</details>

**Comment by alexr00** [comment]
@roblourens since we haven't moved to a unified port forwarding UI yet.

**Comment by roblourens** [comment]
Are you sure your ssh config is not forwarding it? The extension doesn't do anything. automatically.

**Comment by rgwood** [comment]
Nothing in my SSH config, but I can't seem to reproduce this now. This might have been user error, sorry – I'll reopen this with more detail if I can make it happen again.
